test/e2e/antreapolicy_test.go

```diff
@@ -5315,6 +5315,7 @@ func testWithFQDNCacheMinTTL(t *testing.T, fqdnCacheMinTTL int) {
 	)
 
 	skipIfAntreaPolicyDisabled(t)
+	skipIfHasWindowsNodes(t)
 	skipIfNotIPv4Cluster(t)
 	skipIfIPv6Cluster(t)
 	skipIfNotRequired(t, "mode-irrelevant")
```

Comment:
Actually, as part of this PR, do you think you could move all these `skipX` calls to the beginning of `TestFQDNCacheMinTTL`? I think it would be a better place to run these checks, and it would help make this function shorter too.